database/migrations/2020_08_26_163905_create_h5p_counters_table.php

```diff
@@ -15,10 +15,10 @@ public function up()
     {
         Schema::create('h5p_counters', function (Blueprint $table) {
             $table->string('type', 63);
-            $table->unsignedBigInteger('library_id');
-            $table->foreign('library_id')->references('id')->on('h5p_libraries');
+            $table->string('library_name', 127);
+			$table->string('library_version', 31);
             $table->unsignedInteger('num');
-            $table->primary(['type', 'library_id'], 'fk_primary');
+            $table->primary(['type', 'library_name','library_version'], 'fk_primary');
```

Comment:
Add space between library_name and library_version.

Reply:
This one is not fixed yet.

Reply:
added